src/field-group/style.ts

```diff
@@ -1,23 +1,17 @@
 export const FIELD_GROUP_CSS = `
-  kuc-field-group,
-  kuc-field-group *,
-  kuc-field-group:lang(en),
-  kuc-field-group:lang(en) * {
+  kuc-field-group:lang(en) .kuc-field-group__group__toggle .kuc-base-label__text {
```

Comment:
We should add the default setting.
```suggestion
  kuc-field-group .kuc-field-group__group__toggle .kuc-base-label__text,
  kuc-field-group:lang(en) .kuc-field-group__group__toggle .kuc-base-label__text {
```